packages/medusa-plugin-restock-notification/src/services/restock-notification.js

```diff
@@ -117,7 +117,10 @@ class RestockNotificationService extends BaseService {
       }
 
       const variant = await this.productVariantService_.retrieve(variantId)
-      if (variant.inventory_quantity > 0) {
+
+      if (
+        variant.inventory_quantity > (this.options_?.inventory_required ?? 0)
```

Comment:
Does this check also pass, if no options are defined at all? Please add a test

Reply:
there are already tests cases for this possibility. Options are not defined in the other test cases made previous to this PR